frontend/portals/MaintenanceMode/index.jsx

```diff
@@ -1,6 +1,10 @@
 import appConfig from '@shopgate/pwa-common/helpers/config';
 import I18n from '@shopgate/pwa-common/components/I18n';
```

Comment:
I18n can be imported from engage/components too. It can be added to the Link import